### PR TITLE
CI: Add CI configurations to publish frontend build to beamer-bridge/frontend-deployment repo

### PR DESCRIPTION
**Pull request by GabrielBuragev**

### CODE REVIEW

File: .circleci/workflows.yml [hunk 1]
```diff
@@ -297,9 +301,37 @@ jobs:
...
+    working_directory: ~/beamer/frontend
+    environment:
+      REPO_URL: git@github.com:beamer-bridge/frontend-deployment.git
+      TMP_DIR: /tmp/frontend-deployment
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - fa:3a:6e:26:54:04:d8:58:3f:63:7c:3d:00:a1:d7:c2:be
+      - run: |
+          mkdir -p ~/.ssh
```
Comment (panpilkarz) [inline .circleci/workflows.yml:315]: I think `-p` does nothing here and can be removed. Would make sense if there were any subdirectories in `~/.ssh`

Comment (manuelwedler) [reply .circleci/workflows.yml]: Had a failing run without the option

**Comment by manuelwedler** [comment]
Should be ready now

**Comment by GabrielBuragev** [comment]
Note to myself:
After merging this, migrate old mainnet & testnet DO Apps to new ones (including domain pointers)

File: .circleci/workflows.yml [hunk 2]
```diff
@@ -468,10 +468,9 @@ jobs:
 workflows:
   npm-release:
     when: 
-     condition: 
       or: 
-        - <<pipeline.parameters.deployment-artifacts-updated>>
-        - <<pipeline.parameters.deployment-package-json-updated>>
+        - equal: [ true, <<pipeline.parameters.deployment-artifacts-updated>> ]
+        - equal: [ true, <<pipeline.parameters.deployment-package-json-updated>> ]
```
Comment (istankovic) [inline .circleci/workflows.yml:473]: Nitpick: why is `equal` necessary here?
Isn't this the same as the expression it replaces? At least that is my impression from reading https://circleci.com/docs/configuration-reference/#logic-statements

**Review by istankovic — APPROVED** [review]
Beautiful work, @manuelwedler!

**Pull request merged**